### PR TITLE
Start AA version exchange with USB early to avoid headunit timeouts

### PR DESCRIPTION
**Pull request by nisargjhaveri**
Many head units seems to have some timeout for the AA communication to start. Try to work around the timeout by doing the version handshake early with the head unit while we wait for the phone to connect via TCP.

### CODE REVIEW

**Comment by gamelaster** [comment]
When CI will build the artifacts, I will try it. I had like 2-3 times issue that connection timed out.

File: aa_wireless_dongle/package/aawg/src/proxyHandler.cpp [hunk 1]
```diff
@@ -108,7 +111,148 @@ void AAWProxy::forward(ProxyDirection direction, std::atomic<bool>& should_exit)
...
+        return false;
+    }
+
+    return true;
+}
+
+bool AAWProxy::shouldDoEarlyVersionExchange() {
+    // Return false to disable early version exchange
+    return true;
+}
```
Comment (hkfuertes) [inline aa_wireless_dongle/package/aawg/src/proxyHandler.cpp:128]: Nice!

Comment (nisargjhaveri) [reply aa_wireless_dongle/package/aawg/src/proxyHandler.cpp:128]: Well, we need better config mechanism! :D

**Comment by pankajet** [comment]
Is it possible to run standalone android on pi and use Car display for controlling ?

**Comment by nisargjhaveri** [comment]
These changes doesn't seem to help with headunit timeout, at least for me.

**Pull request closed without merging**